### DISCUSSION TITLE
Saving session boolean

### DISCUSSION THREAD

**Discussion by MHKnowles** [BottlecapDave/HomeAssistant-OctopusEnergy#562] [Ideas]
Hi Dave,

Thanks ever so much for your integration - works really well.  However, my coding is almost nonexistent despite owning an HND in Software Engineering (shush!),
Would it therefore be possible for your integration to spit out a boolean field based on whether you've signed up for a saving session so that those of us who don't like to code can have it as a condition in an automation related to dumping our batteries to the grid when it turns on? (i.e. instead of relying solely on the Octoplus Saving Session boolean which might get us dumping our batteries to the grid when we've not actually signed up for the saving session in question).

Thanks muchly!
Mike.

**Comment by BottlecapDave** [comment]
Hello. As per the [docs](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/blob/develop/_docs/entities/octoplus.md#saving-sessions), the current sensor does this

**Comment by MHKnowles** [comment]
Ah, marvellous, sorry Dave I didn’t recognise that was going to happen – thanks!

Mike.

**Comment by BottlecapDave** [comment]
No problem :)
